### PR TITLE
feat(ui): Add `BaseScreen`

### PR DESCRIPTION
**Pull request by wikbro7650**

### CODE REVIEW

**Review by maciejopalinski — APPROVED** [review]

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { render } from '../shared/dom';
+
+const template = `
+    <div>
```
Comment (Veryyapeee) [inline src/components/Header.js]: `<header/>` tag

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { render } from '../shared/dom';
+
+const template = `
+    <div>
```
Comment (Veryyapeee) [inline src/components/Footer.js]: `<footer/>` tag

**Review by Veryyapeee — CHANGES_REQUESTED** [review]
Also please resolve conflicts

File: src/components/Footer.js (hunk 2 above)
Comment (Mariusz-web) [reply src/components/Footer.js]: You can add:
```html
<footer>
  <p>Copyright © 2021 CodersCamp</p>
</footer>
```

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**